src/additional/statesBR.js

```diff
@@ -0,0 +1,30 @@
+/* Validates Brazilian state
+*  Can be case insensitive or require capitalization - default is case insensitive
+*  Can include the Federal District or not - default does not
+*  Usage examples:
+*
+*  This is the default - case insensitive, no include Federal District
+*  stateInput: {
+*     caseSensitive: false,
+*     includeFederalDistrict: false
+*  }
+*
+*  Only allow capital letters
+*  stateInput: {
+*     caseSensitive: true
+*  }
+*
+*/
+$.validator.addMethod("stateBR", function(value, element, options) {
+	var isDefault = typeof options === "undefined",
+		caseSensitive = ( isDefault || typeof options.caseSensitive === "undefined" ) ? false : options.caseSensitive,
+		includeFederalDistrict = ( isDefault || typeof options.includeFederalDistrict === "undefined" ) ? false : options.includeFederalDistrict,
+		regex;
+	if (includeFederalDistrict) {
+		regex = "^(AC|AL|AM|AP|BA|CE|DF|ES|GO|MA|MG|MS|MT|PA|PB|PE|PI|PR|RJ|RN|RO|RR|RS|SC|SE|SP|TO)$";
+    } else {
+		regex = "^(AC|AL|AM|AP|BA|CE|ES|GO|MA|MG|MS|MT|PA|PB|PE|PI|PR|RJ|RN|RO|RR|RS|SC|SE|SP|TO)$";
```

Comment:
I can't see a particular reason to not include DF, it's the Brazil's federal capital, and it's not a state *per si*, but it's always listed as the equivalent for any check-in.